cumulusci/robotframework/Salesforce.py

```diff
@@ -695,6 +724,12 @@ def wait_until_loading_is_complete(self, locator=None):
         try:
             self.selenium.wait_until_page_contains_element(locator)
             self.wait_for_aura()
+            # this knowledge article recommends waiting a second. I don't
+            # like it, but it seems to help. We should do a wait instead,
+            # but I can't figure out what to wait on.
```

Comment:
wait_for_aura should already be waiting for Aura ajax calls to complete (unless they changed the interface for that). I suppose there's still potential for a delay between when they are done and when things have rendered. I wish they exposed a way to get a callback when the event loop is idle.

Reply:
From experimentation, it seems like something is still tweaking the DOM after wait_for_aura returns.

Reply:
I wonder if https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback would be helpful

Reply:
Hmmmm.... interesting idea.  Perhaps we could set a flag in an idle callback, and then our keywords could wait for that flag to be set. That might work, unless the work that the LWCs are doing is also scheduled with that function. 

I don't think I'll try to do that in this PR in the interest of time, but it's definitely an idea worth experimenting on.